test/unit/object_geometry.js

```diff
@@ -218,7 +214,7 @@
     assert.equal(object.containsPoint(point5), false);
   });
 
-  QUnit.test('containsPoint with padding', function(assert) {
+  QUnit.test.skip('containsPoint with padding', function(assert) {
```

Comment:
we should discuss this.
It is not a problem respecting padding in `containsPoint`
I think it should be an option though.